readme.md

```diff
@@ -126,6 +126,14 @@ Run `brew install qlvideo` or [download manually](https://github.com/Marginal/QL
 
 *These are not included in [Install all](#install-all).*
 
+### [Peek](https://bigzlabs.com/peek) 💰
+
+> Peek delivers Quick Look functionality you will not find anywhere else on this planet to over 300 file extensions: copy and find text, jump to line/page numbers, Github-flavored Markdown with a generated table of contents, syntax highlighting, scroll restoration, and more.
```

Comment:
Can you make it less like a marketing tagline? I mean this part "you will not find anywhere else on this planet"